airflow/contrib/hooks/gcs_hook.py

```diff
@@ -284,7 +284,7 @@ def is_updated_after(self, bucket, object, ts):
             storage bucket.
         :type object: str
         :param ts: The timestamp to check against.
-        :type ts: datetime
+        :type ts: datetime.datetime
```

Comment:
```suggestion
        :type ts: datetime.DateTime
```

I think? Similar for other uses and timedelta.

Reply:
timestamp is a specific point in time containing date and time. timedelta is the difference between two timestamp. This is correct for me.

Reply:
For some reason I thought the class was DateTime. Been doing too much of other languages lately.

Reply:
The class name is correct: 
https://docs.python.org/3/library/datetime.html#available-types